joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
 This experimental paradigm --- used across biological modeling, artificial life, and evolutionary computation --- complements research done using *in vitro* and *in vivo* systems by enabling the user to conduct experiments that would be impossible in the lab or field [@dolsonDigitalEvolutionEcology2021].
 One key benefit is complete, exact observability.
 For example, it is possible to perfectly record the full set of parent-child relationships over the history of a population, yielding precise and accurate phylogenies (ancestry trees).
-This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the general factors driving evolution [@dolsonInterpretingTapeLife2020;@moreno2023toward].
+This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the underlying mechanisms driving a given evolutionary system [mooers1997inferring;@dolsonInterpretingTapeLife2020;@moreno2023toward].
 
 The Phylotrack project provides libraries for tracking and analyzing phylogenies in *in silico* evolution.
 The project is composed of 1) Phylotracklib: a header-only C++ library, developed under the umbrella of the Empirical project [@ofria2020empirical], and 2) Phylotrackpy: a Python wrapper around Phylotracklib, created with Pybind11 [@pybind11].
 Both components supply a public-facing API to attach phylogenetic tracking to digital evolution systems, as well as a stand-alone interface for measuring a variety of popular phylogenetic topology metrics [@tuckerGuidePhylogeneticMetrics2017].
-The underlying algorithm design prioritizes efficiency, allowing Phylotrack to support large systems with rapid generational turnover.
-The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that facilitate reducing the memory footprint of phylogenetic information.
+The underlying algorithm design prioritizes efficiency, allowing Phylotrack to support large agent populations with rapid generational turnover.
+The underlying C++ implementation ensures fast, memory-efficient performance, with multiple explicit features (e.g., phylogeny pruning and abstraction, etc.) that can reduce the memory footprint of phylogenetic information.
```

Comment:
Is there a reason you don't like "facilitate reducing"? The idea I was trying to capture is that these features are guaranteed to reduce the memory footprint (which isn't obvious from "can reduce"), but only if the user chooses to use them (which isn't obvious from just saying "reduce")

Reply:
syntactic (not semantic) motivations: destroy gerunds (-ing) and reduce copy count of “facilitate”

that enable further reduction of

Reply:
How about "for reducing"? More succinct and avoids "facilitate" (although it still has the gerund... I might be less anti-gerund than you, though :) )

Reply:
I can buy into that :handshake:  :D